app/src/androidTest/java/org/simple/clinic/TestData.kt

```diff
@@ -186,7 +186,9 @@ class TestData @Inject constructor(
       deletedAt: Instant? = null
   ) = PatientAddress(
       uuid = uuid,
-      colonyOrVillage = colonyOrVilage,
+      houseAddress = null,
```

Comment:
Instead of setting this to `null`, should we make default parameters and set them to specific values?